File: annotation/src/main/java/com/github/snowdream/toybricks/annotation/Interface.java
package com.github.snowdream.toybricks.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by snowdream on 17/2/11.
 */
@Retention(RetentionPolicy.CLASS)
@Target({ElementType.TYPE})
public @interface Interface {
}

File: annotation/src/main/kotlin/com/github/snowdream/toybricks/annotation/Interface.kt
package com.github.snowdream.toybricks.annotation

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * Created by snowdream on 17/2/11.
 */
@Retention(RetentionPolicy.CLASS)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
annotation class Interface
